src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -336,7 +336,7 @@ import os
         /// NOTE: Calling SetupPython multiple times will add the install location to the path many times,
         /// potentially causing the environment variable to overflow.
         /// </summary>
-        private static void InstallPython()
+        internal static void InstallPython()
```

Comment:
I made it internal. Would it be OK to make this public?

Reply:
since it has a side effect that is potentially dangerous I'd prefer if extensions could not call it.